Bio/SCOP/__init__.py

```diff
@@ -467,31 +502,48 @@ def write_cla_sql(self, handle):
         cur = handle.cursor()
 
         cur.execute("DROP TABLE IF EXISTS cla")
-        cur.execute("CREATE TABLE cla (sunid INT, sid CHAR(8), pdbid CHAR(4),\
+        cur.execute(
+            "CREATE TABLE cla (sunid INT, sid CHAR(8), pdbid CHAR(4),\
         residues VARCHAR(50), sccs CHAR(10), cl INT, cf INT, sf INT, fa INT,\
-        dm INT, sp INT, px INT, PRIMARY KEY (sunid), INDEX (SID) )")
+        dm INT, sp INT, px INT, PRIMARY KEY (sunid), INDEX (SID) )"
+        )
 
         for n in self._sidDict.values():
             c = n.toClaRecord()
-            cur.execute("INSERT INTO cla VALUES (%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s)",
-                        (n.sunid, n.sid, c.residues.pdbid, c.residues, n.sccs,
-                         n.getAscendent("cl").sunid, n.getAscendent("cf").sunid,
-                         n.getAscendent("sf").sunid, n.getAscendent("fa").sunid,
-                         n.getAscendent("dm").sunid, n.getAscendent("sp").sunid,
-                         n.sunid))
+            cur.execute(
+                "INSERT INTO cla VALUES (%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s,%s)",
+                (
+                    n.sunid,
+                    n.sid,
+                    c.residues.pdbid,
+                    c.residues,
+                    n.sccs,
+                    n.getAscendent("cl").sunid,
+                    n.getAscendent("cf").sunid,
+                    n.getAscendent("sf").sunid,
+                    n.getAscendent("fa").sunid,
+                    n.getAscendent("dm").sunid,
+                    n.getAscendent("sp").sunid,
+                    n.sunid,
+                ),
+            )
 
     def write_des_sql(self, handle):
         """Write DES data to SQL database."""
         cur = handle.cursor()
 
         cur.execute("DROP TABLE IF EXISTS des")
-        cur.execute("CREATE TABLE des (sunid INT, type CHAR(2), sccs CHAR(10),\
+        cur.execute(
+            "CREATE TABLE des (sunid INT, type CHAR(2), sccs CHAR(10),\
         description VARCHAR(255),\
-        PRIMARY KEY (sunid) )")
+        PRIMARY KEY (sunid) )"
+        )
```

Comment:
Again, same point - I don't see why the original code had internal line continuation slashes in the SQL command.

Reply:
Done.